libeventheader-tracepoint/samples/CMakeLists.txt

```diff
@@ -1,15 +1,24 @@
-add_executable(dynamic-sample
+add_executable(eventheader-dynamic-sample
```

Comment:
Note: All the samples from all the projects go into the same bin folder, so avoid confusion by ensuring that the eventheader samples all start with "eventheader-".